polybar-scripts/system-bluetooth-simple/bluetooth_ON-OFF.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ "$(bluetoothctl show | awk '/Powered/ { print $2 }')" == "no" ]
```

Comment:
include this on/off script in the main script - so that you only have to use one file